### DISCUSSION TITLE
responsiveness on mobile browser

### DISCUSSION THREAD

**Discussion by narasimhajupally** [tailwindlabs/tailwindcss#12078] [Ideas]
In the recent project I have been working on I used h-screen and 100vh etc. which gave the desired output on desktop developer tools responsive testing.
But when I checked the mobile and tablet browsers and discovered that the URL bar is not considered in 100vh and everything was overflowing.
After some searching the fix was to use 100dvh i.e. relative viewport unit.
can we make `h-screen` and other related classes use `dvh` units instead of `vh`?

**Comment by alex-krasikau** [comment]
`100dvh` doesn't equal `100vh` and may produce weird behavior, such as content jumping, when scrolling back and forth.

But I agree, it would be nice to have new viewport units available, perhaps with names like `h-screen-dynamic`, `h-screen-large` and `h-screen-small`.
